src/Components/Facility/Consultations/Mews.tsx

```diff
@@ -0,0 +1,192 @@
+import useQuery from "../../../Utils/request/useQuery";
+import routes from "../../../Redux/api";
+
+export const Mews = (props: { consultationId: string }) => {
+  const { data: dailyRoundsData } = useQuery(routes.getDailyReports, {
+    pathParams: {
+      consultationId: props.consultationId,
+    },
+    query: {
+      rounds_type: "NORMAL,VENTILATOR,ICU",
+      limit: 1,
+    },
+  });
+
+  const respRange = [
+    [-Infinity, 7, 2],
+    [8, 8, 1],
+    [9, 17, 0],
+    [18, 20, 1],
+    [21, 29, 2],
+    [30, Infinity, 3],
+  ];
+
+  const heartRateRange = [
+    [-Infinity, 39, 2],
+    [40, 50, 1],
+    [51, 100, 0],
+    [101, 110, 1],
+    [111, 129, 2],
+    [130, Infinity, 3],
+  ];
+
+  const systolicBloodPressureRange = [
+    [-Infinity, 70, 3],
+    [71, 80, 2],
+    [81, 100, 1],
+    [101, 159, 0],
+    [160, 199, 1],
+    [200, 220, 2],
+    [221, Infinity, 3],
+  ];
+
+  const temperatureRange = [
+    [-Infinity, 94.9, 2],
+    [95, 96.8, 1],
+    [96.9, 100.4, 0],
+    [100.5, 101.3, 1],
+    [101.4, Infinity, 2],
+  ];
```

Comment:
It'd be nice to define this outside the component since these are constants not directly nor indirectly dependent on the props right?